src/main/kotlin/com/ampnet/auditornode/model/contract/ContractAddress.kt

```diff
@@ -0,0 +1,4 @@
+package com.ampnet.auditornode.model.contract
+
+@JvmInline
+value class ContractAddress(val value: String)
```

Comment:
What would be the advantage of wrapping a single value inside a value class like this? Why not just use the value directly?

Reply:
Type safety. Instead of having a `String` as an argument or return value, I can have `ContractAddress` instead, which makes code more readable - you don't have to remember what something is if you can see it in the type name.